src/main/java/in/ac/amu/zhcet/controller/FacultyController.java

```diff
@@ -49,6 +50,7 @@ public String getStudents(Model model, @PathVariable String id) {
         List<CourseRegistration> courseRegistrations = floatedCourse.getCourseRegistrations();
         model.addAttribute("courseRegistrations", courseRegistrations);
         model.addAttribute("course_id", id);
+        model.addAttribute("currentSession", Utils.getCurrentSession());
```

Comment:
Thymeleaf template should automatically access the static method. There is no need to include it in every model